app/src/main/java/com/iron/espresso/presentation/home/mystudy/studydetail/NoticeDetailActivity.kt

```diff
@@ -54,9 +56,21 @@ class NoticeDetailActivity :
                         setBackgroundResource(R.color.colorCobaltBlue)
                     }
                 }
+
             }
         })
 
+        viewModel.toastMessage.observe(this, EventObserver { message ->
+            Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
+        })
+
+        binding.deleteButton.setOnClickListener {
+            viewModel.deleteNotice(studyId, noticeId)
+            setResult(RESULT_OK)
+            onBackPressed()
```

Comment:
'Back 키를 눌렀다' 보다는 '현재화면을 종료시키겠다' 라는 의미가 적합하지 않을까 싶어요
```suggestion
            finish()
```

Reply:
변경하였습니다